org.kde.dolphin.json

```diff
@@ -9,20 +9,25 @@
         "--filesystem=/media",
         "--filesystem=/run/media",
         "--filesystem=home",
+        "--own-name=org.kde.dolphin",
```

Comment:
this is appid which is allowed by default

Reply:
Removed.